### PR TITLE
ONXP-1345 [Assets] Need to sorting by alphabet the assets on all forms

### PR DESCRIPTION
**Pull request by OlehHinko**

### CODE REVIEW

**Comment by OlehHinko** [comment]
@goonerDp, please check

**Comment by goonerDp** [comment]
You have the requirement: "1st asset in the list - oUSD or OnyxCash, other assets are sorted by alphabet "

File: src/api/assets.js [hunk 1]
```diff
@@ -198,3 +198,23 @@ export async function registerSend(values) {
...
+
+export function sortAssetSymbol(assets) {
+	assets.sort(function(a, b) {
+		let nameA = a.symbol.toLowerCase(),
+			nameB = b.symbol.toLowerCase();
+		if (nameA < nameB) return -1;
+		if (nameA > nameB) return 1;
+		return 0;
+	});
+}
```
Comment (goonerDp) [inline src/api/assets.js]: These functions have the same logic, don't duplicate it. Also, return the new array, don't mutate the original.

Comment (goonerDp) [reply src/api/assets.js]: These are helpers and should be at "utils" file.

Comment (goonerDp) [reply src/api/assets.js]: @Oleh, this code has a huge mistake. It's doing unnecessary mapping! Performance is a disaster.

Comment (goonerDp) [reply src/api/assets.js]: Also, sortAsset => **sortAssets**

File: src/api/assets.js [hunk 2]
```diff
@@ -198,3 +198,22 @@ export async function registerSend(values) {
 	});
 	return res.data;
 }
+
+export function sortAssets(assets) {
+	let sortAssets = [];
+	sortAssets = assets.sort(function(a, b) {
+		let nameA;
+		let nameB;
+		console.log("sort");
```
Comment (goonerDp) [inline src/api/assets.js]: Remove this log. And merge into dev.

**Pull request merged**